components/alert/alert.css

```diff
@@ -0,0 +1,31 @@
+/* @import "../../style.css"; */
+@import "../../docSite/docs.css";
+
+
+.alert {
+  max-width: 45rem;
+  padding: 0.5rem 1rem;
+  border-radius: var(--borderRadius);
+}
+
+.alert p {
+  margin: 0;
+}
+.alert-primary {
+  background: var(--clr-blue-primary);
+  color: var(--clr-white);
+}
```

Comment:
Add one line space before defining the next class.  It helps in readability.

Reply:
Thank you for reviewing, I will make the changes!